notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>nit: 2021</p><p> </p><p>Also, overall it wasn't clear to me where the training is happening without looking at your blog post draft. It would help to add a note somewhere in the intro, here in the notebook, where this tutorial is intended to be run - locally? in AI Platform Notebooks?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
https://console.cloud.google.com/ai-platform/notebooks/deploy-notebook?download_url=https://raw.githubusercontent.com/GoogleCloudPlatform/ai-platform-samples/master/notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

You can add this link as a direct "Open in AI Platform Notebooks" link. (Of course, the process after following the link is more straightforward if someone already has their GCP account set up)